components/consentRequestForm/styles.js

```diff
@@ -202,11 +202,17 @@ export default function styles(theme) {
     "agent-name": {
       display: "flex",
       justifyContent: "center",
+      textOverflow: "ellipsis",
```

Comment:
In order for ellipsis to show up, we need to have also `white-space` set to `nowrap`.